src/lib/libXbSymbolDatabase.c

```diff
@@ -1892,8 +1892,14 @@ static bool manual_scan_section_dsound(iXbSymbolContext* pContext,
             internal_RegisterSymbol(pContext, pLibrarySession, XRefNoSaveIndex, 3911,
                 "CDirectSoundStream_GetInfo", *(uint32_t*)(pFuncAddr + 2 * 4));
 
-            internal_RegisterSymbol(pContext, pLibrarySession, XRefNoSaveIndex, 3911,
-                "CDirectSoundStream_GetStatus", *(uint32_t*)(pFuncAddr + 3 * 4));
+            if (pLibrary->build_version < 4134) {
+                internal_RegisterSymbol(pContext, pLibrarySession, XRefNoSaveIndex, 3911,
+                    "CDirectSoundStream_GetStatus__r1", *(uint32_t*)(pFuncAddr + 3 * 4));
```

Comment:
For documentation purposes, maybe it's better to name those symbols by XDK revision of when they first appeared? So `r3911` for this one and `r4134` for the newer one.

Reply:
What is the purpose of 3911 and 4123 numbers you see before the symbol's string name? 😉 If you have use Cxbx-Reloaded, you don't see any XDK build versions in symbol cache files. However, if you set kernel log to console or file from Cxbx-Reloaded, you will see XDK build versions return on first run. Therefore is documented.

In summary, we don't need redundant numbers for each API change. We only need to recognize the function's output has change or the inputs as well. Then apply appropriate expectation for inputs and outputs.